ui/js/treeherder_app.js

```diff
@@ -1,41 +1,18 @@
 import angular from 'angular';
-import ngRoute from 'angular-route';
-import uiBootstrap from 'angular1-ui-bootstrap4';
```

Comment:
Let's also remove `angular1-ui-bootstrap4` from `neutrino-custom/base.js` -- otherwise it gets put in the shared `vendor` chunk (which means since it will mean it still gets downloaded for jobs-view/user-guide/...) when actually perfherder is now the only thing using it.

Reply:
ok, done.  :)